packages/integration-tests/test/standardExitTest.js

```diff
@@ -40,19 +40,19 @@ describe('Standard Exit tests', async () => {
   })
 
   describe('Deposit transaction exit (ci-enabled)', async () => {
-    const INTIIAL_ALICE_AMOUNT = web3.utils.toWei('.1', 'ether')
-    const DEPOSIT_AMOUNT = web3.utils.toWei('.0001', 'ether')
+    const INTIIAL_ALICE_AMOUNT = web3.utils.toWei('1', 'ether')
```

Comment:
Why has this value been increased? It'll break the test runner I think.

Reply:
no reason, random fiddling to try fix something we thought broken haha will revert